.github/workflows/tests.yml

```diff
@@ -38,6 +38,7 @@ jobs:
       run: python3 -m pytest -s -vvv tests/unit/
 
     - name: Execute integration-tests
+      timeout-minutes: 10
```

Comment:
I introduced a bug on the fixtures that made the tests run forever, without this the build runs for hours:

![image](https://user-images.githubusercontent.com/310139/163040970-613d8513-b858-4d34-95a1-c93bb8461c47.png)

Reply:
This should be also on the unit tests.

Reply:
done